### ISSUE TITLE
Tetra patches get numbered from 0, instead of 1

### ISSUE THREAD

**Issue by Andy2No** [christofmuc/KnobKraft-orm#161]
The DSI Tetra displays patch numbers from 1 to 128, but if I import a bank into Orm, it numbers them from 0 to 127.   It would be nice if they could match.

**Comment by christofmuc** [comment]
@Andy2No That is true, the GenericSequential adpatation, which the Tetra is based on, does not implement the friendly bank name or friendly program name methods yet. I think this is because the numbering varies wildly across all the sequential synths. I'll look into it to see if that can be implemented on a per-synth basis. What are the banks called on the Tetra?

**Comment by Andy2No** [comment]
The Program mode banks are just numbered, 1 to 4, and the patches in them are numbered 1 to 128.

I briefly experimented with my own version of the adaptation, because I couldn't get the official one to work with the firmware I had - at least not via USB.  I can't remember if I tried it with DIN MIDI.  I  needed to send some things twice, with that firmware (1.6.6, I think) - especially the device detection and the the first patch request in a bank.  I settled temporarily for just sending the request twice each time.

I had this, which I think worked okay:

def numberOfBanks():
    return 4 # numbered 1 to 4


def numberOfPatchesPerBank():
    return 128 # numbered 1-128


def friendlyProgramName(program):
    return str(program+1)
